### PR TITLE
modified madrid_air_air_quality to accept command line parameters

### PR DESCRIPTION
**Pull request by Joudsi**
modified the air quality harvester to accept parameter -  
example : python madrid_air_quality_harvest.py 28079059 --service fiwareExample --service-path / --endpoint http://localhost:1026 --latest

@caa06d9c 
@jmcanterafonseca

### CODE REVIEW

File: specs/Environment/AirQualityObserved/harvest/barcelona_airquality_harvest.py [hunk 1]
```diff
@@ -207,6 +211,8 @@ def get_air_quality_barcelona(target_stations):
 
     # Now persisting data to Orion Context Broker
     for a_station in entity_data:
+        if a_station not in stations_to_retrieve_data:
```
Comment (caa06d9c) [inline specs/Environment/AirQualityObserved/harvest/barcelona_airquality_harvest.py]: If no station defined, this check will never pass. By default, if no station defined, all available stations should be used.

File: specs/Environment/AirQualityObserved/harvest/barcelona_airquality_harvest.py [hunk 2]
```diff
@@ -325,6 +336,47 @@ def setup_logger():
...
+    parser.add_argument('--endpoint', metavar='endpoint', type=str, nargs=1,
+                        help='Context Broker end point. '
+                             'Example. http://orion:1030')
+    parser.add_argument('--latest', action='store_true',
+                        help='Flag to indicate to only '
+                             'harvest the latest observation')
+
+    args = parser.parse_args()
+
+    if args.service:
```
Comment (caa06d9c) [inline specs/Environment/AirQualityObserved/harvest/barcelona_airquality_harvest.py]: Default values should be defined, or Orion will return an error.

File: specs/Environment/AirQualityObserved/harvest/barcelona_airquality_harvest.py [hunk 3]
```diff
@@ -325,6 +336,47 @@ def setup_logger():
...
+        FIWARE_SERVICE = args.service[0]
```
Comment (caa06d9c) [inline specs/Environment/AirQualityObserved/harvest/barcelona_airquality_harvest.py]: Default values should be defined, or Orion will return an error.

**Comment by caa06d9c** [comment]
@jmcanterafonseca i found that some stations return errors:
> WARNING 2018-07-26 18:45:24,899 No data found for station: 08019004
Maybe it's better to update list?

**Comment by jmcanterafonseca** [comment]
@caa06d9c 08019004 is an station code from Barcelona not from Madrid (it starts with `08`). 
Madrid's stations start with `28`

**Pull request merged**